junit5/src/main/java/com/linecorp/armeria/testing/junit5/client/HttpHeadersAssert.java

```diff
@@ -0,0 +1,193 @@
+        assertTrue(actual().containsObject(name, value));
+        return response();
+    }
+
+    /**
+     * Asserts that the actual {@link HttpHeaders} contains the given name and boolean value.
+     * The {@code name} cannot be null.
+     */
+    public TestHttpResponse containsBoolean(CharSequence name, boolean value) {
+        requireNonNull(name, "name");
+        assertTrue(actual().containsBoolean(name, value));
+        return response();
+    }
+
+    /**
+     * Asserts that the actual {@link HttpHeaders} contains the given name and int value.
+     * The {@code name} cannot be null.
+     */
+    public TestHttpResponse containsInt(CharSequence name, int value) {
+        requireNonNull(name, "name");
+        assertTrue(actual().containsInt(name, value));
+        return response();
+    }
+
+    /**
+     * Asserts that the actual {@link HttpHeaders} contains the given name and long value.
+     * The {@code name} cannot be null.
+     */
+    public TestHttpResponse containsLong(CharSequence name, long value) {
+        requireNonNull(name, "name");
+        assertTrue(actual().containsLong(name, value));
+        return response();
+    }
+
+    /**
+     * Asserts that the actual {@link HttpHeaders} contains the given name and float value.
+     * The {@code name} cannot be null.
+     */
+    public TestHttpResponse containsFloat(CharSequence name, float value) {
+        requireNonNull(name, "name");
+        assertTrue(actual().containsFloat(name, value));
+        return response();
+    }
+
+    /**
+     * Asserts that the actual {@link HttpHeaders} contains the given name and double value.
+     * The {@code name} cannot be null.
+     */
+    public TestHttpResponse containsDouble(CharSequence name, double value) {
+        requireNonNull(name, "name");
+        assertTrue(actual().containsDouble(name, value));
+        return response();
+    }
+
+    /**
+     * Asserts that the actual {@link HttpHeaders} contains the given name and time(millis) value.
+     * The {@code name} cannot be null.
+     */
+    public TestHttpResponse containsTimeMillis(CharSequence name, long value) {
+        requireNonNull(name, "name");
+        assertTrue(actual().containsTimeMillis(name, value));
+        return response();
+    }
+
+    /**
+     * Asserts that the size of actual {@link HttpHeaders} is equal to the given one.
+     */
+    public TestHttpResponse sizeIsEqualTo(int expected) {
+        assertEquals(expected, actual().size());
+        return response();
+    }
+
+    /**
+     * Asserts that the actual {@link HttpHeaders} is empty.
+     */
+    public TestHttpResponse isEmpty() {
+        assertTrue(actual().isEmpty());
+        return response();
+    }
+
+    /**
+     * Asserts that the names of actual {@link HttpHeaders} contains the given ascii string.
+     * The {@code expected} cannot be null.
+     */
+    public TestHttpResponse namesContains(AsciiString expected) {
```

Comment:
General suggestion: what do you think of referring to `assertj` for naming rules in general? - this would give a better way to sync rules and is probably more comfortable for armeria developers in general.

```suggestion
    public TestHttpResponse containsName(AsciiString expected) {
```